server/src/routes/api/orgs.js

```diff
@@ -8,65 +8,62 @@ import { getMcsoRecords } from '../../services/scraper';
 import moment from 'moment-timezone';
 
 export default Router()
-  .get('/:orgId', requireAuth, confirmOrg, (req, res, next) => {
-    const { orgId } = req.params;
-    Org.findById(orgId)
+  .get('/', requireAuth, (req, res, next) => {
```

Comment:
Now that i think about it, the `confirmOrg` middleware isn't necessary. Just use the user.org everywhere and remove the orgId from the url path